<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdCenterFinderDialog</class>
 <widget class="QDockWidget" name="QxrdCenterFinderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Centering</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="margin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QToolButton" name="m_CenterMoveUpLeft">
           <property name="toolTip">
            <string>Move Center Up and Left</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/go-up-left.png</normaloff>:/images/go-up-left.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="m_CenterMoveUp">
           <property name="toolTip">
            <string>Move Center Up</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/go-up.png</normaloff>:/images/go-up.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="m_CenterMoveUpRight">
           <property name="toolTip">
            <string>Move Center Up and Right</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/go-up-right.png</normaloff>:/images/go-up-right.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="m_CenterMoveLeft">
           <property name="toolTip">
            <string>Move Center Left</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/go-left.png</normaloff>:/images/go-left.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_13">
           <property name="toolTip">
            <string>Color wheel - if beam is not properly centered
find the color on the right of the group of peaks
and click the corresponding button to move the
center in the right direction</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="qxrdresources.qrc">:/images/color-wheel.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="m_CenterMoveRight">
           <property name="toolTip">
            <string>Move Center Right</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/go-right.png</normaloff>:/images/go-right.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="m_CenterMoveDownLeft">
           <property name="toolTip">
            <string>Move Center Down and Left</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/go-down-left.png</normaloff>:/images/go-down-left.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="m_CenterMoveDown">
           <property name="toolTip">
            <string>Move Center Down</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/go-down.png</normaloff>:/images/go-down.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="m_CenterMoveDownRight">
           <property name="toolTip">
            <string>Move Center Down and Right</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/go-down-right.png</normaloff>:/images/go-down-right.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>X Center:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="m_CenterX">
        <property name="toolTip">
         <string>X Center in pixels
script property centering.centerX</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Y Center:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="m_CenterY">
        <property name="toolTip">
         <string>Y Center in Pixels
script property centering.centerY</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Step Size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QxrdCenterStepSpinner" name="m_CenterStep">
        <property name="toolTip">
         <string>Step Size for Center moves
script property centering.centerStep</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>X Pixel Size:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="m_DetectorXPixelSize">
        <property name="toolTip">
         <string>Detector X Pixel Size (micrometer)
script property centering.detectorXPixelSize</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Y Pixel Size:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="m_DetectorYPixelSize">
        <property name="toolTip">
         <string>Detector Y Pixel Size (micrometer)
script property centering.detectorYPixelSize</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Detector Distance</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="m_DetectorDistance">
        <property name="toolTip">
         <string>Sample-Detector Distance (mm)
script property centering.detectorDistance</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QCheckBox" name="m_ImplementTilt">
        <property name="toolTip">
         <string>Enable Advanced Tilt Geometry?
script property centering.implementTilt</string>
        </property>
        <property name="text">
         <string>Implement Tilt</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Tilt Angle</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="m_DetectorTilt">
        <property name="toolTip">
         <string>Tilt angle of detector plane (deg)
script property centering.detectorTiltTilt</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>90.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Tilt Plane Rotation</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QDoubleSpinBox" name="m_TiltPlaneRotation">
        <property name="toolTip">
         <string>Rotation of tilt plane (deg) 0=horizontal
script property centering.tiltPlaneRotation</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Photon Energy(eV)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QDoubleSpinBox" name="m_Energy">
        <property name="toolTip">
         <string>Photon Energy (eV)
script property centering.energy</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QCheckBox" name="m_EnableGeometry">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Enable Geometry Corrections </string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Polarization Factor</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QDoubleSpinBox" name="m_Polarization">
        <property name="toolTip">
         <string/>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QDoubleSpinBox" name="m_Absorption">
        <property name="toolTip">
         <string/>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Attenuation Coefficient</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QCheckBox" name="m_EnablePolarization">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Enable Polarization Corrections </string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QCheckBox" name="m_EnableAbsorption">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Enable Absorption Corrections </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxrdCenterStepSpinner</class>
   <extends>QDoubleSpinBox</extends>
   <header>qxrdcenterstepspinner.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qxrdresources.qrc"/>
 </resources>
 <connections/>
</ui>
